roles/nuage_node/tasks/main.yaml

```diff
@@ -33,6 +33,19 @@
 
 - include: certificates.yml
 
+- name: Add additional Docker mounts for Nuage for atomic hosts
+  become: yes
+  lineinfile: dest="{{ openshift_atomic_node_config_file }}" line="{{ nuage_atomic_docker_additional_mounts }}" insertafter='(.*)IMAGE_VERSION(.*)'
+  when: openshift.common.is_atomic | bool
+
+- name: reload daemon and restart atomic-openshift-node service for Atomic install
+  command: /bin/true
+  notify:
+    - reload daemon
```

Comment:
this can be removed if `daemon-reload` flag is used with ansible systemd module.